framework/include/problems/FEProblemBase.h

```diff
@@ -2318,6 +2318,9 @@ class FEProblemBase : public SubProblem, public Restartable
   virtual void setCurrentLowerDElem(const Elem * const lower_d_elem, const THREAD_ID tid) override;
   virtual void setCurrentBoundaryID(BoundaryID bid, const THREAD_ID tid) override;
 
+  // Used for setting previous states
+  int global_current_state;
```

Comment:
```suggestion
  int _global_current_state;
```
we prefix member variables withe an underscore